ui/components/component-library/modal-content/modal-content.stories.tsx

```diff
@@ -185,11 +179,10 @@ export const Size: StoryFn<typeof ModalContent> = (args) => {
                 )}
               </Text>
             </ModalBody>
-            <Box padding={4}>
-              <Button onClick={() => setCurrentSize(null)} block>
-                Close
-              </Button>
-            </Box>
+            <ModalFooter
+              onSubmit={() => setCurrentSize(null)}
+              submitButtonProps={{ children: 'Close' }}
+            />
           </ModalContent>
         </Modal>
       )}
```

Comment:
Replacing placeholders in `ModalContent` stories and docs with `ModalFooter`

### Before

https://github.com/MetaMask/metamask-extension/assets/8112138/ada03463-ce54-4995-9cff-80fa29f802bd



### After


https://github.com/MetaMask/metamask-extension/assets/8112138/3b4c68d5-f2da-4618-ad5e-eb63df628f6d